src/CTA.WebForms2Blazor/ControlConverters/ControlConverter.cs

```diff
@@ -64,10 +113,48 @@ public static HtmlDocument ConvertEmbeddedCode(HtmlDocument document)
 
             //Not implemented/used yet
             //htmlString = EmbeddedCodeReplacers.AspExpRegex.Replace(htmlString, aspExprEval);
+            
+            return htmlString;
+        }
+        
+        //This function only updates the first child node that matches the name,
```

Comment:
This function looks like it updates every node that matches the name, not just the first. Is this comment outdated?

Reply:
Yep, I'll change that